ecc_logic.py

```diff
@@ -291,7 +291,12 @@ def get_appearance_gender(appearance):
 
 def get_value_for_key_and_id_in_appearance(appearance, idx, key):
     """ Loops through the appearance json to match a dictionary with id = idx and then returns the value of ['key'] """
+
     storables = appearance['storables']
+

```

Comment:
I'm not in favor of such obfuscated code. I do mind an extra line if it improves readability and makes it clear what's going on.